src/tds/query.c

```diff
@@ -728,12 +728,48 @@ tds_get_column_declaration(TDSSOCKET * tds, TDSCOLUMN * curcol, char *out)
 	size = tds_fix_column_size(tds, curcol);
 
 	switch (tds_get_conversion_type(curcol->on_server.column_type, curcol->on_server.column_size)) {
-	case XSYBCHAR:
+	case XSYBCHAR: //same as SYBLONGCHAR
+		if (!IS_TDS7_PLUS(tds->conn)) { //SYBLONGCHAR
```

Comment:
I think this would be better to be `IS_TDS50(tds->conn)`.

Reply:
I agree. Thanks.

Reply:
Fixed